std/cs/internal/HxObject.hx

```diff
@@ -276,7 +276,7 @@ class HxEnum {
 	}
 
 	public function getTag():String {
-		return throw 'Not Implemented';
+		return throw new haxe.exceptions.NotImplementedException('Not Implemented');
```

Comment:
Why specify the message in these cases, seems you made it optional with "Not Implemented" as the default?

Reply:
Indeed. Updated.